apps/web/src/app/[locale]/explore/_components/explore.action.ts

```diff
@@ -30,6 +32,22 @@ export async function getChallengesByTagOrDifficulty(str: string, take?: number)
           }),
     },
     include: {
+      ...( session && session?.user?.id ? {
```

Comment:
i dont think you need the `session &&` since youre using optional chaining

Reply:
Changed ! I thought the id was optional on a user